### ISSUE TITLE
Click-to-fill bounding box

### ISSUE THREAD

**Issue by txau** [huridocs/uwazidocs#4878]
**Is your feature request related to a problem? Please describe.**
Currently users don't know if they have selection rectangles for a particular value. Also they don't have a visual cue of the current selected area nor can delete it.

**Describe the solution you'd like**
- When the user inputted the value via click-to-fill it needs to show the bounding box of the selection rectangles. That way users have a visual way of knowing if the selection is correct, particularly if they decide to change the value afterwards and the new value is not coming from the same area.
- This bounding box should have some interaction button to delete it, removing the selection rectangles from the file.
- Make sure that the selection rectangle and the label data remain even if you modify the value manually.

This will be developed in #4883
- Extra ball: we give users some UI option to navigate to stored selection rectangles when interacting with that particular field. This functionality already exists in the side-panel document overlay in the machine learning interface. It would be great to extend it to the regular document viewer for Uwazi.

**Comment by Zasa-san** [comment]
For the initial implementation of this feature the button to clear the selected text will be in the entity form, right below the properties that have selections.

**Comment by fnocetti** [comment]
@Zasa-san I think the idea of this part:

> - This bounding box should have some interaction button to delete it, removing the selection rectangles from the file.

is that you can delete the selection rectangle and it's labeled data from the file, not necessarily clear the value from the field.
As it is conceptualized right now, the selection rectangles + labeled data are a self contained piece of information that is linked to the value in the field at the moment of using the click-to-fill functionality, but once finished, we don't want neither:
- Have manually editing the field value remove the selection rectangles, nor
- Removing the selection rectangles clear the value in the field. 
That's more or less stated here:
> - Make sure that the selection rectangle and the label data remain even if you modify the value manually.

In other words, the delete button described in the issue is related to the selection rectangle, not the field. That's why I think that having the button near the field may be misleading. 

I would like @txau to confirm this.

**Comment by Zasa-san** [comment]
To clarify: The button will only clear the highlighted text in the PDF and the information from the file, it will not remove anything from the field or the entity's metadata.

I agree that a better scenario would be to have the button near the selected text, to signify that the actions are clearly performed on the selection and not the metadata of the entity. Right now our Highlight component cannot do that. 
I think the next step in this feature should be to enhance our Highlight component to handle not only a button, but also display a container box around the selection, and the label of the property it's linked to.

**Comment by txau** [comment]
@fnocetti @Zasa-san from your comments I think a good solution could be having the "Delete selection" button rendered in the selection itself. We can also involve @juanmnl to rethink the whole click-to-fill and selection removal UX.
